tools/sigmut/sigmut_macros.xml

```diff
@@ -0,0 +1,23 @@
+<macros>
+      
+    <token name="@VERSION@">11.19</token>
```

Comment:
I don't understand this VERSION variable. What is it ?

Reply:
At the beginning I used as version ID the month + year of development. In this case, 11.19 means that this version was created in November 2019.

Reply:
this is not best practice for version values at all...

Reply:
No, I guess not. Therefore, it will be '0.9.0' from now on.